app/pages/ServiceDiscoveryForm/ServiceDiscoveryForm.jsx

```diff
@@ -99,6 +100,15 @@ const InnerServiceDiscoveryForm = ({
               .map(c => c.name),
           },
         };
+
+        const categoriesRefinements = searchState?.refinementList?.categories?.join('; ') || 'NONE';
+        const eligibilitiesRefinements = searchState?.refinementList?.eligibilities?.join('; ') || 'NONE';
```

Comment:
Yeaaaaa I think I was being excessively paranoid considering this file hadn't been converted to TS yet, and I didn't know how defensive the ST conventions are when that's the case. Anyhow, I'll just remove the operators since it's trivial to do so.

Reply:
Your paranoia is justified, because there's a lot of places where we're not null-safe and where we do often see null or undefined errors in Sentry. I think it's gotten better in the past year, a lot due to @lexholden's work on porting files over to TypeScript, so hopefully we get through the whole codebase soon!